src/HomeWork1.java

```diff
@@ -0,0 +1,52 @@
+public class HomeWork1 {
+
+
+    public static void printThreeWords () {
+        System.out.println("Orange");
+        System.out.println("Banana");
+        System.out.println("Apple");
+    }
+
+
+    public static void checkSumSign () {
+        int a = 10;
+        int b = 7;
+        int sum = a + b;
+        if (sum >=0) {
+            System.out.println("Сумма положительна");
+        } else {
+            System.out.println("Сумма отрицательна");
+        }
+    }
+
+
+    public static void printColor() {
+        int value = 17;
+        if (value <= 0) {
+            System.out.println("Красный");
+        } else if (value > 0 && value <= 100) {
```

Comment:
```suggestion
        } else if (value <= 100) {
```
поскольку условие выше уже исключает данную проверку на 0